src/assets/img/icons/invite-users.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Would be good if this was 24x24 like all other icons.

Reply:
okay :)

Reply:
This is so that all icons have predicable positioning no matter how big they are inside of the 24x24 box

Reply:
@PascalPrecht so what is the best way to do it? Should I change just view box?

Reply:
Usually, the designer has prepared all icons in the designs so that they are surrounded by such a box and can then be exported that way.

Maybe you can reach out to the designer in charge and ask the same.